python/ql/src/semmle/python/frameworks/Django.qll

```diff
@@ -1530,11 +1671,62 @@ private module Django {
     result = djangoRouteHandlerFunctionTracker(DataFlow::TypeTracker::end(), func)
   }
 
+  /** A django View class defined in project code. */
+  class DjangoViewClassDef extends Class {
+    DjangoViewClassDef() { this.getABase() = django::views::generic::View::subclassRef().asExpr() }
+
+    /** Gets a function that could handle incoming requests, if any. */
+    DjangoRouteHandler getARouteHandler() {
+      // TODO: This doesn't handle attribute assignment. Should be OK, but analysis is not as complete as with
+      // points-to and `.lookup`, which would handle `post = my_post_handler` inside class def
```

Comment:
Is this a case where type backtracking would help?

Reply:
Proper MRO is really what I want here :) (I created internal ticket for this)

type backtracking might help with implementing MRO, but that's an adventure for an other day :blush: